pallets/gear-program/src/benchmarking.rs

```diff
@@ -44,7 +44,8 @@ benchmarks! {
         let code = benchmarking::generate_wasm(q.into()).unwrap();
 
         let program_id = ProgramId::from_origin(benchmarking::account::<T::AccountId>("program", 0, 100).into_origin());
-        benchmarking::set_program(program_id.into_origin(), code, q.into());
+
+        benchmarking::set_program(program_id.into_origin(), code, q.into(), 0);
```

Comment:
this will be removed once benchmarking fn adapted

Reply:
Done